pallets/common/src/benchs/asset.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{
+    benchs::User,
+    constants::currency::POLY,
+    traits::{
+        asset::{AssetFnTrait, AssetName, AssetType},
+        identity::Trait as IdentityTrait,
+    },
+};
+
+use polymesh_primitives::{ticker::TICKER_LEN, Ticker};
+
+use frame_system::RawOrigin;
+use sp_std::{convert::TryFrom, vec};
+
+/// Create a ticker and register it.
+pub fn make_ticker<Asset, Balance, Acc, O, N>(owner: O, opt_name: Option<N>) -> Ticker
+where
+    Asset: AssetFnTrait<Balance, Acc, O>,
+    N: AsRef<[u8]>,
+{
+    let ticker_name = match &opt_name {
+        Some(name) => name.as_ref(),
+        _ => [b'A'; TICKER_LEN as usize].as_ref(),
+    };
+    let ticker = Ticker::try_from(ticker_name).unwrap();
+    Asset::register_ticker(owner, ticker).unwrap();
```

Comment:
If the above comment is acceptable, this becomes something like that:
```suggestion
    let ticker = Ticker::try_from(ticker_name)?;
    Asset::register_ticker(owner, ticker)?;
```

Reply:
Related with previous comment.